yellowbrick/classifier/prcurve.py

```diff
@@ -327,46 +351,51 @@ def _draw_binary(self):
         """
         Draw the precision-recall curves in the binary case
         """
-        self._draw_pr_curve(self.recall_, self.precision_, label="binary PR curve")
+        self._draw_pr_curve(self.recall_, self.precision_, label="Binary PR curve")
         self._draw_ap_score(self.score_)
 
     def _draw_multiclass(self):
         """
         Draw the precision-recall curves in the multiclass case
         """
-        # TODO: handle colors better with a mapping and user input
         if self.per_class:
+
+            colors = dict(zip(self.classes_, self._colors))
+
             for cls in self.classes_:
                 precision = self.precision_[cls]
                 recall = self.recall_[cls]
 
                 label = "PR for class {} (area={:0.2f})".format(cls, self.score_[cls])
-                self._draw_pr_curve(recall, precision, label=label)
+                self._draw_pr_curve(recall, precision, label=label, color=colors[cls])
 
-        if self.micro:
+        elif self.micro:
```

Comment:
Does this make sense? It occurred to me that it was at least possible that a user might set *both* `micro` and `per_class` to true...

Reply:
That's a good catch! I suggest to also add a warning for this case to let the user know that `micro=True` is ignored if `per_class` is also set to true.